webpack/mainnet.config.js

```diff
@@ -29,12 +30,7 @@ module.exports = {
     }),
     new webpack.optimize.OccurrenceOrderPlugin(),
     new webpack.IgnorePlugin(/[^/]+\/[\S]+.dev$/),
-    new webpack.optimize.UglifyJsPlugin({
-      comments: false,
-      compressor: {
-        warnings: false
-      }
-    }),
+    new MinifyPlugin(),
```

Comment:
do you know if this plugin deletes the comments?

Reply:
It definitely had an option for this, but I'm not sure if it's enabled by default, tho. Will look into it!